app/webpack/javascripts/modules/Modules.MultiFileUpload.js

```diff
@@ -1,12 +1,65 @@
-/* global MOJFrontend */
+/* global MOJFrontend, FormData, XMLHttpRequest */
 
 moj.Modules.MultiFileUpload = {
   init: function () {
     const container = document.querySelector('.moj-multi-file-upload')
     if (!container) return
     const fields = container.querySelector('.moj-multi-file__uploaded-fields')
+    const formIdElement = document.querySelector('#claim_form_id')
+    let formId = null
 
-    return new MOJFrontend.MultiFileUpload({
+    if (formIdElement) {
+      formId = formIdElement.value
+    }
+
+    const uploadFile = function (file) {
+      this.params.uploadFileEntryHook(this, file)
+      const formData = new FormData()
+      formData.append('document[document]', file)
+
+      if (formId) {
+        formData.append('document[form_id]', formId)
+      }
+
```

Comment:
Ideally we do not really want to overwrite what comes from moj-framework as this could result in missing any updates or bug fixes. However, I appreciate that it does not behave as we need it to at the moment.

Perhaps it is worth having;

```suggestion
      this.params.uploadFileParamsHook(this, formData)
```

and then, further down at the definition of `upload`, adding: 

```javascript
      uploadFileParamsHook: function (_uploader, formData) {
        const formIdElement = document.querySelector('#claim_form_id')
        if (formIdElement) {
          formData.append('document[form_id]', formIdElement.value)
        }
      },
```

This then gives us the option of submitting a feature request and/or a pull request to have this hook added.